ia/src/game.py

```
@@ -0,0 +1,45 @@
+import process
+import protocole
+import socket
+import threading
+
+threads = []
+clients = []
+
+def add_client(client) -> None : 
```

Ruff (0.11.9)

Missing type annotation for function argument `client`

(ANN001)